pkg/ensurance/grpcc/connection.go

```diff
@@ -0,0 +1,56 @@
+package grpcc
+
+import (
+	"fmt"
+	"time"
+
+	"google.golang.org/grpc"
+	"k8s.io/kubernetes/pkg/kubelet/util"
+
+	"github.com/gocrane/crane/pkg/utils/log"
+)
+
+const (
+	defaultTimeout = 2 * time.Second
+)
+
+func InitGrpcConnection(endPoints []string) (*grpc.ClientConn, error) {
```

Comment:
is there an library in kubernetes to call runtime or we need to maintain our own.